src/TransactionTest.java

```diff
@@ -0,0 +1,77 @@
+import org.junit.Test;
+
+import java.util.Date;
+
+import static org.junit.Assert.*;
+
+public class TransactionTest {
+
+    @Test
+    public void givenNoDataOrMoney_returnsNoDataOrMoney() {
+        String summary = new Transaction().getSummary();
```

Comment:
REFACTOR: Since we're only concerned about the 'blank', default Transaction here let's just create one Transaction, and reuse it, instead of calling `new Transaction()` repeatedly. Applies to multiple tests.